app/models/email.rb

```diff
@@ -0,0 +1,26 @@
+class Email < ApplicationRecord
+  belongs_to :notification
+  validates :subject, :body, :notification, presence: true
+
+  def self.create_from_params!(params)
+    build_from_params(params).tap(&:save!)
```

Comment:
Do we need the `tap` here? Can we not just do `...(params).save!`?

Reply:
It's so as the method returns the instance. If we just do `.save!` it returns `true` or raises.

Reply:
Ah yep, makes sense.